presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -0,0 +1,197 @@
+=================
+Iceberg Connector
+=================
+
+Overview
+--------
+
+Apache Iceberg is an open table format for huge analytic datasets.
+The Iceberg connector allows querying data stored in
+files written in Iceberg format, as defined in the
+`Iceberg Table Spec <https://iceberg.apache.org/spec/>`_.
+
+The Iceberg connector supports:
+
+* All Presto types, including TIME, TIMESTAMP, DECIMAL
```

Comment:
This isn't accurate as there are many Presto types not supported by Iceberg (for example, Geospatial data types)

Reply:
Removed in the previous edit.